grisera_api/graph_api_service.py

```diff
@@ -0,0 +1,41 @@
+import requests
```

Comment:
I suppose that this import is not needed anymore.

Reply:
I think you meant in participant_service.py 😅

Reply:
Sure I did :)